src/pages/Library.tsx

```diff
@@ -313,68 +708,41 @@ class Library extends React.Component<ILibraryProps, ILibraryState>
-                                    curNum = parseInt(curNum);
-                                }
-
-                                //This is the next button, so we wanna go to next chapter (+ check edge cases)
-                                if(curNum == 22){
-                                    curNum = 25;
-                                }else if(curNum == 29){
-                                    //Do nothing, we're at the end. Maybe put a nice message saying that you've reached the end?
-                                }else{
-                                    //Not an edge case, just advance
-                                    curNum++;
-                                }
-
-                                //Turn to chapter curNum
-                                this.setChapter(curNum);
-                            }}
-                        >
-                            <IonIcon icon = {ellipsisVerticalOutline}></IonIcon>
-                        </IonButton>
-                        
+                    {/*/!* Animated chapter navigation buttons (grey) *!/*/}
+                    {/*/!* DevNote: this is the code for making it work w/o animations : style={this.state.showArrows ? {} : { display: 'none' }}*!/*/}
+                    
+                    <Slide right when={this.state.showArrows}>
+                        <div id="right-arrow" className="nav-arrow" onClick={()=>{
+                            this.nextChapter(); 
+                            }}>
+                                <IonIcon id="chev-right" className="chev-nav-buttons" style={this.state.last ? {display:"none"} : {}} icon={chevronForwardOutline}></IonIcon>
+                        </div>
+                    </Slide>
+                    <Slide left when={this.state.showArrows}>
+                        <div id="left-arrow" className="nav-arrow" onClick={()=>{this.prevChapter();}}>
+                            <IonIcon id="chev-left" className="chev-nav-buttons" style={this.state.last ? {display:"none"} : {}} icon={chevronBackOutline}></IonIcon>
+                        </div>
+                    </Slide>
+                    
+                       <div style={this.state.showFab ? translucent : translucentGone}>                   {/*/!* Div needed to allow opacity <1; Fade overrides it to 1 :(. Must be completely "gone" to not block elements beneath*!/*/}
+                        <Fade when={this.state.showFab}>
+                            <div id="fab" style={this.state.last ? beginning : {}} onClick={()=>{
+                                this.nextChapter(); 
+                            }}>
+                                <div id="div-next-text" style={this.state.context.Mode == "statement" || this.state.last ? {} : {fontSize:"12px"}}>Next:</div>
+                                <div id="div-proverbs-text" style={this.state.context.Mode == "statement" || this.state.last ? {} : {fontSize:"16px"}}>P</div>
```

Comment:
Instead of changing this with InnerHTML, do it the react way with state.

Reply:
React is not made to have components updated with both state and InnerHTML edits. React's guiding philosophy is that the component is rendered solely based on state.